content/uk/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -103,7 +103,7 @@ <h2 style="color: #3771e3;">Як розгорнути ваш перший зас
 
                 <!--<p>When you create a Deployment, you'll need to specify the container image for your application and the number of replicas that you want to run. You can change that information later by updating your Deployment; Modules <a href="/docs/tutorials/kubernetes-basics/scale-intro/">5</a> and <a href="/docs/tutorials/kubernetes-basics/update-intro/">6</a> of the bootcamp discuss how you can scale and update your Deployments.</p>
                 -->
-                <p>Коли ви створюєте Deployment, вам необхідно задати образ контейнера для вашого застосунку і скільки реплік ви хочете запустити. Згодом цю інформацію можна змінити, оновивши Deployment. У навчальних модулях <a href="/docs/tutorials/kubernetes-basics/scale-intro/">5</a> і <a href="/docs/tutorials/kubernetes-basics/update-intro/">6</a> йдеться про те, як масштабувати і оновлювати Deployment'и.</p>
+                <p>Коли ви створюєте Deployment, вам необхідно задати образ контейнера для вашого застосунку і скільки реплік ви хочете запустити. Згодом цю інформацію можна змінити, оновивши Deployment. У навчальних модулях <a href="/uk/docs/tutorials/kubernetes-basics/scale/scale-intro/">5</a> і <a href="/uk/docs/tutorials/kubernetes-basics/update/update-intro/">6</a> йдеться про те, як масштабувати і оновлювати Deployment'и.</p>
```

Comment:
Додавання цих елементів шляху - виправлення чи обдруківка?
```suggestion
                <p>Коли ви створюєте Deployment, вам необхідно задати образ контейнера для вашого застосунку і скільки реплік ви хочете запустити. Згодом цю інформацію можна змінити, оновивши Deployment. У навчальних модулях <a href="/uk/docs/tutorials/kubernetes-basics/scale-intro/">5</a> і <a href="/uk/docs/tutorials/kubernetes-basics/update-intro/">6</a> йдеться про те, як масштабувати і оновлювати Deployment'и.</p>
```

Reply:
You can take a look at #20316, which covers exactly this issue.